code/datums/greyscale/greyscale_configs.dm

```diff
@@ -126,3 +126,16 @@
 /datum/greyscale_config/carp/disk_mouth
 	name = "Space Carp Disk in Mouth"
 	json_config = 'code/datums/greyscale/json_configs/carp_disk_mouth.json'
+
+/datum/greyscale_config/janicart_upgrade
+	name = "Janicart Upgrade"
+	icon_file = 'monkestation/icons/obj/janitor/janicart_upgrade.dmi'
+	json_config = 'code/datums/greyscale/json_configs/janicart_upgrade.json'
+
+/datum/greyscale_config/janicart_upgrade/installed
+	json_config = 'code/datums/greyscale/json_configs/janicart_upgrade_installed.json'
```

Comment:
Unrelated change?

Reply:
I was just moving them out of the modular folder since they were a complete icon replacement for the janicart attachments

Reply:
No issues with the move

![image](https://user-images.githubusercontent.com/101475356/184942998-edd1dac2-317c-4431-8d1d-d6588f471ca5.png)